core/Scheduler/Scheduler.php

```diff
@@ -121,6 +131,11 @@ public function run()
 
             // loop through each task
             foreach ($tasks as $task) {
+                if (in_array($this->signal, [\SIGINT, \SIGTERM], true)) {
+                    $this->logger->info("Scheduler: Aborting due to received signal");
```

Comment:
Would it be helpful to log the signal that is received, SIGINT or SIGTERM

Reply:
Might not hurt, but it also doesn't make a difference between the signals here.
Also this is currently only triggered through core archiving, and that one prints the signal once it receives one: https://github.com/matomo-org/matomo/pull/22487/files#diff-f41c275e883996b8b69cb765b1818e12e82cbe034e648bc3f7d03610cc7abad1R317